### ISSUE TITLE
Design Spike: Accessible Confirmation of Delete Actions

### ISSUE THREAD

**Issue by kellylein** [department-of-veterans-affairs/va-mobile-app#4696]
In our work with the Nav and IA update we found that the Snack Bar we use for delete action confirmation could be improved upon to make it more accessible and lightweight.  We should look at a larger pattern change to account for this.  
From a Slack comment from Holly:
> we can do better in terms of solving the larger/global pattern issue of accessible confirmation of delete actions by including subheader text—that reflects the change—for the screenreader to shift focus to in order to confirm the change on the screen without adding a extra UI element (that would be a separate ticket)

**AC:**
- [ ] Review app for similar situations to take into account when forming our solution: [VA Mobile App - Screen Roundup](https://app.mural.co/invitation/mural/adhoccorporateworkspace2583/1644341290076?sender=u28718b63c8993f515e0b2240&key=74759cd5-45ad-43d9-aee3-a58fb949aa73)
- [ ] Explore & determine an alternative pattern for delete action confirmation that is more accessible & works as a global treatment ([preliminary options](https://www.figma.com/file/B3cMgZxUlawJhbK7ltDbco/Lauren's-Sandbox?node-id=708%3A3566&t=XQH3P4lcF7CPh3wr-4) – can evolve)
- [ ] Create mockup(s) for Eng

**Comment by timwright12** [comment]
@jessicawoodin @kellylein design system?

**Comment by jessicawoodin** [comment]
@timwright12 Yup, I think this can be a design system ticket that we'll tackle with both the components and pattern documentation.

**Comment by timwright12** [comment]
@kellylein can you migrate this ticket over and close it out when you have a moment?
